tests/lib/sprintf/src/main.c

```diff
@@ -935,17 +937,18 @@ void test_fwrite(void)
 	zassert_equal(ret, 0, "fwrite failed!");
 }
 
+
 /**
  *
  * @brief Test stdout_hook_default() function
  *
  * @details When CONFIG_STDOUT_CONSOLE=n the default
  * stdout hook function _stdout_hook_default() returns EOF.
  */
-void test_EOF(void)
+#else
+ZTEST(sprintf, test_EOF)
 {
 	int ret;
-
```

Comment:
If this line removed you might meet a compliance check error. Keep it pls.

Reply:
Thank You Enjia for reviewing it!! Pushed all the suggested changes.